src/bindings/jna/libelektra/src/main/java/org/libelektra/Elektra.java

```diff
@@ -354,6 +360,25 @@ int elektraGOptsContractFromStrings (Pointer contractKeySet, long argsSize, Stri
 
 	@Nullable Pointer ksAtCursor (Pointer ks, int cursor);
 
+	/**
+	 * Flag for use with {@link #ksLookup(Pointer, Pointer, int)} and
+	 * {@link #ksLookupByName(Pointer, String, int)}. Represents no option set.
+	 */
+	static final int KDB_O_NONE = 0;
+
+	/**
+	 * Flag for use with {@link #ksLookup(Pointer, Pointer, int)}. Will cause the
+	 * release of parameter {@code key} using {@link #keyDel(Pointer)}.
+	 */
+	static final int KDB_O_DEL = 1;
```

Comment:
From the release notes I understood that this was removed, not added?

Reply:
It has been removed from the Java binding's public API. But since it is still part of Elektra's `ksLookup` API, it has been moved to the Elektra native library proxy interface (which i will make package private with the next commit ;))